app/controllers/planners_controller.rb

```diff
@@ -15,6 +15,15 @@ def create
     end
   end
 
+  def show
+    @planner = Planner.find(params[:id])
+    @reservable_tables = @planner.reservable_tables
```

Comment:
これだと過去もでてしまうので、当日以前のものは出ないようにしたいです！
```suggestion
    @reservable_tables = @planner.reservable_tables.where('date >= ?', Date.current)
```

Reply:
あとソートもしてほしいです！
ソートは当日に近い順がいいと思うので、
```
.order(date: :asc)
```
を後ろにつけてあげると良さそうです

Reply:
対応しました。